core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/error/ErrorRepository.java

```diff
@@ -0,0 +1,27 @@
+package org.silverpeas.core.workflow.engine.error;
+
+import org.silverpeas.core.persistence.datasource.repository.jpa.BasicJpaEntityRepository;
+import org.silverpeas.core.persistence.datasource.repository.jpa.NamedParameters;
+import org.silverpeas.core.util.SilverpeasList;
+
+import java.util.List;
+
+/**
+ * Created by Nicolas on 07/06/2017.
+ */
+public class ErrorRepository extends BasicJpaEntityRepository<WorkflowErrorImpl> {
+
+  public List<WorkflowErrorImpl> getByProcessInstanceId(String id) {
+    NamedParameters parameters = newNamedParameters();
+    parameters.add("id", Integer.parseInt(id));
+    SilverpeasList<WorkflowErrorImpl>
+        errors = listFromNamedQuery("processInstance.findErrors", parameters);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "errors". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)